cpg-neo4j/src/main/kotlin/de/fraunhofer/aisec/cpg_vis_neo4j/Application.kt

```diff
@@ -589,6 +611,13 @@ class Application : Callable<Int> {
     }
 }
 
+class AstChildrenEventListener : EventListenerAdapter() {
```

Comment:
Is this a merge relict? Didn't we get rid of this and use a pass instead?

Reply:
Possible, however I don't remember when we deleted this in the main branch :/

Reply:
I think you can safely remove it